ts/features/pushNotifications/store/selectors/notificationsBannerDismissed.ts

```diff
@@ -0,0 +1,30 @@
+import { getTime } from "date-fns";
+import { GlobalState } from "../../../../store/reducers/types";
+import { messageListForCategorySelector } from "../../../messages/store/reducers/allPaginated";
+import { UIMessage } from "../../../messages/types";
+
+const NEW_MESSAGES_COUNT_TO_RESET_FORCE_DISMISS = 4;
+
+export const timesPushNotificationBannerDismissedSelector = (
+  state: GlobalState
+) => state.notifications.userBehaviour.pushNotificationsBanner.timesDismissed;
+
+export const shouldResetNotificationBannerDismissStateSelector = (
+  state: GlobalState
+) => {
+  const forceDismissDate =
+    state.notifications.userBehaviour.pushNotificationsBanner
+      .forceDismissionDate;
+  const messagesList = messageListForCategorySelector(state, "INBOX");
```

Comment:
This selector was made to cover the visual aspects of the inbox/archive list in the MessagesHome screen. Inspecting it, we can see that `none` states are treated differently (undefined vs empty list) and that is not what we want here (since we want to operate on the data only when it is `some`.

I think that at this point is better to either:
- create a new custom selector that returns the array of messages directly (using category as input), or
- directly retrieve the data that we need here, like `const inboxMessageListOrUndefined = pot.toUndefined(state.entities.messages.allPaginated.inbox.data)?.page`